src/net/sourceforge/kolmafia/textui/command/ColdMedicineCabinetCommand.java

```diff
@@ -290,56 +289,60 @@ public void run(final String cmd, String parameter) {
       case "booze", "wine" -> collect(3);
       case "potion" -> collect(4);
       case "pill" -> collect(5);
+      case "pills" -> guessTurnsRequiredForPills();
```

Comment:
Can we come up with a different param? I think being so close to `pill` is not good UX. Also this reminds me that we haven't added any of the params to the help string - not your fault but would appreciate if you could do it at least for your new param and we can fill in the other missing ones that predate your changes.